test/functional-tests/include/Config.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include "TmpFile.hpp"
+
+#include <string>
+#include <map>
+
+class Config {
+public:
+    Config(std::string instances = "",
```

Comment:
"instances": that not clear what it is. If I understand the code above correctly, it is the toplevel configuration. But if I didn't read the code above, I would have thought this is the content of `<InstanceDefinition>`.